spec/unit/inspection_multiple_spec.rb

```diff
@@ -166,11 +166,15 @@
 end
 
 describe 'aasm.states_for_select' do
-  it "should return a select friendly array of states" do
-    expect(FooMultiple.aasm(:left)).to respond_to(:states_for_select)
-    expect(FooMultiple.aasm(:left).states_for_select).to eq(
-      [['Open', 'open'], ['Closed', 'closed'], ['Final', 'final']]
-    )
+  context 'without I18n' do
```

Comment:
These change is necessary because of reason described there https://github.com/aasm/aasm/pull/689/files#r457542825